packages/grafana-ui/src/components/Combobox/MultiCombobox.tsx

```diff
@@ -284,6 +287,23 @@ export const MultiCombobox = <T extends string | number>(props: MultiComboboxPro
           />
 
           <div className={multiStyles.suffix} ref={suffixMeasureRef} {...getToggleButtonProps()}>
+            {isClearable && selectedItems.length > 0 && (
+              <Icon
```

Comment:
Does this work properly when `getToggleButtonProps()` is spread on the parent? The suffix is absolutely positioned, which is handled through padding with auto sizing, so either make this conditionally add more padding, or place it outside of the suffix div.

Reply:
After @joshhunt comments, I was increasing the padding-right when the `Clear all` button is shown. The code should be changed now.